repositories/ActivityRepository.ts

```diff
@@ -30,13 +30,30 @@ export class ActivityRepository extends BaseRepository<ActivityModel> {
     const activity = {
       user,
       type,
-      description,
       pointsEarned,
+      description,
       scope: ActivityRepository.activityScopes[type],
     };
     return this.repository.save(ActivityModel.create(activity));
   }
 
+  public async batchLogActivity(
+    users: UserModel[], types: ActivityType[], pointsEarnedBatch?: number[], descriptionBatch?: string[],
+  ): Promise<ActivityModel[]> {
+    const activities: ActivityModel[] = [];
```

Comment:
My thinking was to have `batchLogActivity` match `logActivity` with arrays, but yeah it ends up being less clean. I added an internal `Activity` and `Attendance` type to pass in, should we also change the arguments of `logActivity` and `attendEvent` to take in that type rather than multiple args?